ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-rhods-rhoam__v4.14-stage.yaml

```diff
@@ -0,0 +1,62 @@
+base_images:
+  cli:
+    name: "4.14"
+    namespace: ocp
+    tag: cli
+  ocp-addons-operators-cli:
+    name: ocp-addons-operators-cli
+    namespace: ci
+    tag: latest
+  rhods-addon-tests:
+    name: ods-ci
+    namespace: ci
+    tag: latest
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: ci
+      version: "4.14"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: co-exist-rosa-rhods-rhoam
+  cron: '@yearly'
+  steps:
+    cluster_profile: aws-managed-cspi-qe
+    env:
+      ADDON1_CONFIG: name=managed-odh;notification-email=interop-qe-ms@redhat.com
+      ADDON2_CONFIG: name=managed-api-service;addon-managed-api-service=1;rosa-cli-required=true;notification-email=interop-qe-ms@redhat.com;rosa=true
+      FIREWATCH_CONFIG: |
+        [
+          {"step": "addon-install", "failure_type": "all", "classification": "Provisioning - Addons", "jira_project": "MSINTEROP", "jira_epic": "MSINTEROP-380"},
+          {"step": "addon-uninstall", "failure_type": "all", "classification": "Deprovisioning - Addons", "jira_project": "MSINTEROP", "jira_epic": "MSINTEROP-380"},
+          {"step": "addon-tests-*", "failure_type": "all", "classification": "Addons tests", "jira_project": "RHODS", "jira_epic": "RHODS-7870"},
```

Comment:
please update changes according to https://github.com/openshift/release/pull/42304/ and  https://github.com/openshift/release/pull/42106

Reply:
Done